.gitignore

```diff
@@ -6,6 +6,9 @@ coverage/
 .config/
 .v8flags*
 
+# for storing windbreaker secrets - e.g. shared github hooks secrets
+.secrets/
```

Comment:
Open for suggestion on how we store/access secrets - I just went with this approach to get things working initially.